### PR TITLE
Updating to latest swagger and regenerating code.

### PR DESCRIPTION
**Pull request by vinagesh**
I have taken the latest swagger provided by the IoTHub team and regenerated our code. I've removed the workaround we had to ignore build errors due to missing xml comments. Everything builds and works now.

Changes in new swagger
1) All descriptions are updated
2) Non-GA fields/features are removed
3) Classification of clients has been done as we requested

### CODE REVIEW

File: sdk/iot/Azure.Iot.Hub.Service/src/Azure.Iot.Hub.Service.csproj [hunk 1]
```diff
@@ -3,8 +3,6 @@
     <AssemblyTitle>Azure IoT Hub Service Client SDK</AssemblyTitle>
     <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
     <EnableApiCompat>false</EnableApiCompat>
```
Comment (vinagesh) [inline sdk/iot/Azure.Iot.Hub.Service/src/Azure.Iot.Hub.Service.csproj:5]: This is no longer needed. Yay :)

File: sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs [hunk 2]
```diff
@@ -192,9 +196,9 @@ public virtual Task<Response<BulkRegistryOperationResponse>> CreateIdentitiesWit
                                 : ExportImportDeviceStatus.Enabled,
                     StatusReason = x.Key.StatusReason,
                     ImportMode = ExportImportDeviceImportMode.Create
-                }.WithTags(x.Value.Tags).WithPropertiesFrom(x.Value.Properties).WithParentScopes(x.Key.ParentScopes));
```
Comment (abhipsaMisra) [inline sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs:195]: what about parent scope? was it removed from swagger?

Comment (vinagesh) [reply sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs:195]: yes it has been removed as it is not GA

File: sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs [hunk 3]
```diff
@@ -144,11 +143,23 @@ private AccessToken GetSasTokenAsync(CancellationToken cancellationToken)
             }
         }
 
+        /// <summary>
+        /// Gets an Azure.Core.AccessToken for the specified set of scopes.
```
Comment (drwill-ms) [inline sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs:147]: `<see cref="Azure.Core.AccessToken"/>`

File: sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs [hunk 4]
```diff
@@ -144,11 +143,23 @@ private AccessToken GetSasTokenAsync(CancellationToken cancellationToken)
...
+        /// </summary>
+        /// <param name="requestContext">The Azure.Core.TokenRequestContext with authentication information.</param>
+        /// <param name="cancellationToken">The System.Threading.CancellationToken to use.</param>
+        /// <returns>     A valid Azure.Core.AccessToken.</returns>
```
Comment (drwill-ms) [inline sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs:151]: Why extra space after the returns start tag?

File: sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs (hunk 2 above)
Comment (drwill-ms) [reply sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs:195]: What do you mean "as it is not GA"?

**Review by abhipsaMisra — APPROVED** [review]

File: sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs (hunk 4 above)
Comment (vinagesh) [reply sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs:151]: There is no need. Will fix the spacing.

File: sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs (hunk 3 above)
Comment (vinagesh) [reply sdk/iot/Azure.Iot.Hub.Service/src/Authentication/IotHubSasCredential.cs:147]: I will add it here. But we need a task to update almost all comments and add goodness to our C# track 2 SDK.

**Review by drwill-ms — APPROVED** [review]

File: sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs (hunk 2 above)
Comment (vinagesh) [reply sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs:195]: They were adding new properties that are not used in any GA APIs. They are intended to be used by a preview feature which is not released yet.

**Review by bikamani — APPROVED** [review]

File: sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs (hunk 2 above)
Comment (drwill-ms) [reply sdk/iot/Azure.Iot.Hub.Service/src/DevicesClient.cs:195]: gotcha

**Pull request merged**